docs/conceptual/runtime/primitives.md

```diff
@@ -71,4 +71,4 @@ There are an additional set of primitives that are assumed about a runtime built
 ### References
 
 - View the [`traits` defined in
-  `sr-primitives`](https://substrate.dev/rustdocs/master/sr_primitives/traits/index.html).
+  `node-primitives`](https://substrate.dev/rustdocs/master/node_primitives/traits/index.html).
```

Comment:
This link is still broken .Maybe you meant to use https://substrate.dev/rustdocs/master/sp_runtime/traits/index.html ?

Reply:
Agree, I think that the correct one should be `sp-runtime`  https://substrate.dev/rustdocs/master/sp_runtime/traits/index.html